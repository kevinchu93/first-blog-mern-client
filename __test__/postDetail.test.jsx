import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import PostDetail from '../src/app/postDetail';

test('PostDetail component should render as expected', () => {
  expect(toJson(shallow(<PostDetail />))).toMatchSnapshot();
});

