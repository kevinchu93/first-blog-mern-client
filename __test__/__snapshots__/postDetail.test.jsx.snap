// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostDetail - shallow PostDetail component should render as expected - snapshot 1`] = `
<div>
  <h2>
    testtitle
  </h2>
  <p>
    testbody
  </p>
  <p
    id="author"
  >
    testAuthor
  </p>
</div>
`;
